### ISSUE TITLE
no attribute 'load_kwargs' if you use JSONFormField

### ISSUE THREAD

**Issue by brigantino2** [bradjasper/django-jsonfield#139]
If I try to use `JSONFormField` as a field for one of my forms, I run into an

```
AttributeError
'JSONFormField' object has no attribute 'load_kwargs'
```

in fact `load_kwargs` is initialized in model's field `JSONFieldBase`, but not in the form's field.

**Comment by kazmiruk** [comment]
My PR https://github.com/bradjasper/django-jsonfield/pull/129 should fix it

**Comment by dmkoch** [comment]
PR #129 should be merged in version 2.0.0 on PyPI. Please reopen this issue if there is still a problem. Thanks!
